src/lms-ds-loader/edfi_lms_ds_loader/migrator.py

```diff
@@ -53,57 +34,45 @@ def _read_statements_from_file(full_path: str) -> List[str]:
     with open(full_path) as f:
         raw_sql = f.read()
 
-    return split(raw_sql)
+    statements: List[str] = split(raw_sql)
+    return statements
 
 
-def _execute_statements(engine: Engine, statements: List[str]):
-    def __callback(session: sa_session):
-        for statement in statements:
-            # Ignore MSSQL "GO" statements
-            if statement == "GO":
-                continue
-
-            # Deliberately throwing away all results. Counting on exception handling
-            # if there are any errors, and migration scripts should not be returning
-            # any results.
-            session.execute(statement)  # type: ignore
-
-    _execute_transaction(engine, __callback)
```

Comment:
Also moved the above function to `sql_adapter.py`. I had hoped that doing so would help me with unit testing, but no luck so far.